app/views/shared/_recently_updated_documents_from_rummager.html.erb

```diff
@@ -0,0 +1,13 @@
+<ol class="document-list">
```

Comment:
Same applies here about the using document list component if possible. If not, removing the `h2`.